ivy/functional/frontends/numpy/manipulation_routines/changing_array_shape.py

```diff
@@ -9,6 +9,9 @@
 def reshape(x, /, newshape, order="C"):
     return ivy.reshape(x, shape=newshape, order=order)
 
+@to_ivy_arrays_and_back
+def compress(condition, x, axis=None, out=None):
```

Comment:
ivy functional API doesn't have this function you will need to implement it using composition of other ivy functions

Reply:
there is no extract function to mask the array while compressing. do i have to implement extract function too?